recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
-{% set version = "0.6" %}
-{% set sha256 = "edcf3e908f0d873e569af593166416affa6e87e08bedb8bddca5f50fc9799f2f" %}
-{% set number = "1" %}
+{% set version = "0.7.1" %}
+{% set sha256 = "c7744df945fa90791ad9b4183a6a7dc8220d63a7b8a5f8f93ba62086f1e69e83" %}
+{% set number = "0" %}
 
 package:
   name: python-graphviz
   version: {{ version }}
 
 source:
-  fn: python-graphviz-{{ version }}.tar.gz
-  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
+  fn: graphviz-{{ version }}.zip
```

Comment:
-1: this is the 'offifical' name of the source distribution file (would you rename the file on download when building by hand?).